addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3651,6 +3646,14 @@ var SnippetsMenu = Widget.extend({
             }
         }
     },
+    /**
+     * Hook for website.
+     *
+     * @returns {bool} - false
+     */
+    _isMobile() {
+        return false;
```

Comment:
Would not it make sense to call the web_editor util you made actually? :shrug: It will return "false" in all supported usecase anyway but even when it returns "true", it won't hurt. It will actually be consistent... this code about invisible snippets does not make sense for mass mailing anyway (for now?).